ceph/ceph_admin/bootstrap.py

```diff
@@ -353,6 +356,7 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
                     cmd = "cephadm shell --"
                     cmd += f" ceph config set mgr mgr/cephadm/container_image_{image}"
                     cmd += f" {override_dict[image_key]}"
+                    logger.info(f"Executing command : {cmd}")
```

Comment:
I think `exec_command` will add log record with detail, maybe we can take this out.

Reply:
Done. removed